tests/common.py

```diff
@@ -258,8 +258,8 @@ def get_da_cred(issuer, schema, registry):
 
     return creder   
 
-def get_ecr_auth_cred(aid, issuer, recipient, schema, registry, sedge, lei: str):
-    sad = dict(get_ecr_data(lei))
+def get_ecr_auth_cred(aid, issuer, recipient, schema, registry, sedge, lei: str, role="EBA Data Submitter"):
```

Comment:
Would prefer this to be a constant here and in the other param-defaults

Reply:
Should I define it at the start of the commons.py file as DEFAULT_EBA_ROLE constant and use it in all the parameters?

Reply:
That's fine with me for now. Ideally we will consolidate to a single common file that lives in src, so that BOTH the src and test files can use the DEFAULT_ECR_ROLE constant, so that we aren't defining it in multiple spots.

Reply:
Got it. I already implemented it in places. Let me do that here as well.

Reply:
Changes made in the latest commit.